lib/active_admin_import/dsl.rb

```diff
@@ -76,7 +76,8 @@ def active_admin_import(options = {}, &block)
             instance_exec result, options, &DEFAULT_RESULT_PROC
           end
         rescue ActiveRecord::Import::MissingColumnError, NoMethodError, ActiveRecord::StatementInvalid => e
-          flash[:error] = I18n.t('active_admin_import.file_error', message: e.message)
+          Rails.error.log(I18n.t('active_admin_import.file_error', message: e.message))
+          flash[:error] = I18n.t('active_admin_import.file_error', message: e.message[0..200])
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.